.github/workflows/bsim-tests.yaml

```diff
@@ -40,8 +40,9 @@ jobs:
       BSIM_OUT_PATH: /opt/bsim/
       BSIM_COMPONENTS_PATH: /opt/bsim/components
       EDTT_PATH: ../tools/edtt
-      bsim_bluetooth_test_results_file: ./bsim_bluetooth/bsim_results.xml
-      bsim_networking_test_results_file: ./bsim_net/bsim_results.xml
+      bsim_bluetooth_test_results_file: ./bsim_bluetooth/52_bsim_results.xml
+      bsim_bluetooth_53_test_results_file: ./bsim_bluetooth/53_bsim_results.xml
+      bsim_networking_test_results_file: ./bsim_net/52_bsim_results.xml
```

Comment:
What about the `bsim_bluetooth_test_results_file` and `bsim_networking_test_results_file`? Should we be renamed to `bsim_bluetooth_52_test_results_file` and `bsim_networking_52_test_results_file`?

Reply:
I can even shorten the names for free ;).